### ISSUE TITLE
feature suggestion: new field type: "editable HTML"

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2977]
new feature suggestion:

new field type "editable HTML".

In edit view of a module, this field would show up with a WYSIWYG editor (including option to directly edit raw HTML).

Use cases e.g.:

- field "description" in module TemplateSectionLine
- in combination with new functionality in AOS_PDF_Templates: When including a field of type "editable HTML", the HTML code would be interpreted in PDF generation, instead of printing it as code (text).

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [feature suggestion: new field type: "editable HTML"](https://trello.com/c/zdt6sawh/39-feature-suggestion-new-field-type-editable-html)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
